### ISSUE TITLE
Factsheets can't connect to DB

### ISSUE THREAD

**Issue by LunaUrsa** [TripSit/tripmobile#48]

**Comment by tacerus** [comment]
Originally it was the Combo section, if the Factsheet section isn't working now as well we'd really need someone to look into the source code to see what it tries to connect to.

**Comment by LunaUrsa** [comment]
This particular issue is probably a result of the URL needing to be HTTPS. 

CHanged the DrugNameAsyncTask.java line from:

private static final String ALL_DRUGS_URL = "http://tripbot.tripsit.me/api/tripsit/getAllDrugNames";

To:

private static final String ALL_DRUGS_URL = "https://tripbot.tripsit.me/api/tripsit/getAllDrugNames";

**Comment by tacerus** [comment]
Can we apply the same solution to #49 ?

**Comment by LunaUrsa** [comment]
I don't believe so, but i will take a look

**Comment by LunaUrsa** [comment]
Pushed my branch up, commit for this fix is here: https://github.com/TripSit/tripmobile/commit/aa6f3a49348bd3d0f2a924b05b8f0ffc92d54ab1

I dont know github well enough to shortcode that =D

**Comment by LunaUrsa** [comment]
Yeah it turned out to be the same issue. Why it would crash in combos but not factsheets is weird but okay, pushed up my commit
